### ISSUE TITLE
Get information about an At-rule

### ISSUE THREAD

**Issue by peterbe** [csstree/csstree#57]
I convert a CSS string to an AST, then to a plain object. Now I want to decide which At-rule to keep. In particular I want to do different things if it's a print rule. I.e. I want to distinguish between media queries and print queries. 

```
> text=`@media print { a { color: red} }
... @media (min-width:100px) { a { color: blue} }
... `
'@media print { a { color: red} }\n@media (min-width:100px) { a { color: blue} }\n'
> csstree.parse(text).children
List {
  cursor: null,
  head:
   { prev: null,
     next: { prev: [Circular], next: null, data: [Object] },
     data:
      { type: 'Atrule',
        loc: null,
        name: 'media',
        expression: [Object],
        block: [Object] } },
  tail:
   { prev: { prev: null, next: [Circular], data: [Object] },
     next: null,
     data:
      { type: 'Atrule',
        loc: null,
        name: 'media',
        expression: [Object],
        block: [Object] } } }
```

There is no information in each of these the values `print` or `(min-width:100px)`.

**Comment by lahmatiy** [comment]
It's not clear to me what information do you miss.
At-rule prelude can be found in `Atrule.expression`. For your example (I removed some redundant parts from output):
```
> console.log(JSON.stringify(csstree.parse(text), null, 4))
{
    "type": "StyleSheet",
    "children": [
        {
            "type": "Atrule",
            "name": "media",
            "expression": {
                "type": "AtruleExpression",
                "children": [
                    {
                        "type": "MediaQueryList",
                        "children": [
                            {
                                "type": "MediaQuery",
                                "children": [
                                    {
                                        "type": "Identifier",
                                        "name": "print"
                                    }
                                ]
                            }
                        ]
                    }
                ]
            },
            "block": { ... }
        },
        {
            "type": "Atrule",
            "name": "media",
            "expression": {
                "type": "AtruleExpression",
                "children": [
                    {
                        "type": "MediaQueryList",
                        "children": [
                            {
                                "type": "MediaQuery",
                                "children": [
                                    {
                                        "type": "MediaFeature",
                                        "name": "min-width",
                                        "value": {
                                            "type": "Dimension",
                                            "value": "100",
                                            "unit": "px"
                                        }
                                    }
                                ]
                            }
                        ]
                    }
                ]
            },
            "block": { ... }
        }
    ]
}
```
So you need search for a `MediaQuery` nodes and search for a `Identifier` nodes with the name `print` in their children.

**Comment by peterbe** [comment]
ah! My bad. I should have realized it's a list and thus I should have dug deeper. And I also forgot to use the `JSON.stringify` instead of console to see both objects.

So basically, it's there. Just need to look deeper. Then I guess we can resolve this. 

For my application I might just wait for the next release of csstree to get that new `Atrule.prelude` key.

**Comment by lahmatiy** [comment]
> I might just wait for the next release of csstree to get that new Atrule.prelude key
It's already shipped, see [v1.0.0-alpha23](https://github.com/csstree/csstree/releases/tag/v1.0.0-alpha23)
